graphqlmetrics/cmd/main.go

```diff
@@ -149,7 +156,7 @@ func main() {
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		ms.Shutdown(cfg.ShutdownDelay)
```

Comment:
Why did you remove it?

Reply:
Initially I removed the pool and wanted to ensure to process everything in the batch processor. It does make sense to have a timeout here. I will re-add it.